backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/shipment/M_InOut_StepDef.java

```diff
@@ -132,15 +141,19 @@ public class M_InOut_StepDef
 	private final IInputDataSourceDAO inputDataSourceDAO = Services.get(IInputDataSourceDAO.class);
 	private final IHUInOutBL huInOutBL = Services.get(IHUInOutBL.class);
 	private final IMsgBL msgBL = Services.get(IMsgBL.class);
+	private final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
 
 	@And("^validate the created (shipments|material receipt)$")
 	public void validate_created_shipments(@NonNull final String inoutType, @NonNull final DataTable table)
 	{
 		final List<Map<String, String>> dataTable = table.asMaps();
 		for (final Map<String, String> row : dataTable)
 		{
+			logger.info("validate_created_shipment: {}", row);
```

Comment:
so now that u modified the whole method, why not change it and use DataTableRows instead?

Reply:
because they dont exist here https://github.com/metasfresh/metasfresh/blob/inner_silence_uat/backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/DataTableRows.java